ibis/__init__.py

```diff
@@ -71,6 +72,12 @@
     # pip install ibis-framework[bigquery]
     import ibis.bigquery.api as bigquery
 
+with suppress(ImportError):
+    # pip install ibis-framework[mapd]
+    if sys.version_info[0] < 3:
+        raise ImportError('ibis.mapd is not allowed it for Python 2.')
```

Comment:
typo here, should read `ibis.mapd is not allowed for Python 2` or `The mapd backend is not supported under Python 2`

Reply:
thanks!